### DISCUSSION TITLE
Device without IP

### DISCUSSION THREAD

**Discussion by Bellavista** [make-all/tuya-local#1319] [Q&A]
Good morning everyone.
I have some room thermometers connected to the IoT fine, I can read all the characteristics, only one problem.
They are not connected via IP. If I ping my network, they are not visible. I can recover the Mac via the app.
The question is.... can I set up them with Tuya-local?
Thank

**Comment by make-all** [comment]
Do you mean you have Bluetooth devices, and are using them directly from the Tuya app?  To make these available locally, you will need to use a hub.
